### ISSUE TITLE
Texture Touch-up

### ISSUE THREAD

**Issue by T145** [T145/magistics#60]
Many people have been telling me that the old TC textures are a bit stale; and while nostalgia is powerful I can finally see what is meant. Therefore, if there's any artist who's interested in contributing, please leave a post or PM me!

The plan is to have a style reminiscent of Elucent's, using predominate gold and silver color scheme. Both corresponding metals are typically associated w/ magical arts, so I feel it fits the Magistics theme, and will help differentiate the mod further. I want to keep the patterns generally the same, but everything else (including models) is open to reinterpretation. For the Dark Infuser I'd like black, gold and purple as it's color scheme, and other similar "dark arts" devices or apparatus.

**Comment by Boundarybreaker** [comment]
I've made a new version of the Dark Infuser based on the request. I've shortened the infuser height by four pixels, and am working on a new standard Infuser to match it as well. Let me know what you think, and if it needs any changes! You can see the whole commit [here](https://github.com/Boundarybreaker/magistics/commit/3a8e9629c9425aa5c9b3e5ee1c63105efa72bc56).
![](https://cdn.discordapp.com/attachments/290916642232336394/383542946928132106/image.png)
![](https://cdn.discordapp.com/attachments/290916642232336394/383543325577183233/image.png)
![](https://cdn.discordapp.com/attachments/290916642232336394/383543156395737099/image.png)

**Comment by T145** [comment]
I like it! My only recommendation for a change would be to fix those side connections. I also think you could extend the purple all the way down to meet at the bottom and then have the connections put over the purple. Then for the normal Infuser you could have a similar design using white and gray; replace the black w/ some lighter gray and the purple w/ some white or lighter gray. You could also just have the gold extend down the sides for both infusers and kind of fade away so it's not so blocky. I'd also try to change the yellow to more of a gold color. Right now it kind of looks like the yellow used on Mekansim's glowstone tools and armor.

Use the image below for reference as to my desired color palate:

![mechanism](https://user-images.githubusercontent.com/1214129/33219949-32db298c-d113-11e7-9aa9-3e7db576a01e.png)

**Comment by Boundarybreaker** [comment]
I dunno how I feel about putting the connections over the purple, because I thought of the purple as a cloth, like the red is on the enchanting table. That's why I made it part over the connection; because it's getting pushed out of the way by a pipe. I can definitely try out putting the connection over it, and I'll recolor the gold.

**Comment by Boundarybreaker** [comment]
Here's a few modifications to the design based on what you described. This more like what you want?
![2017-11-24_22 11 45](https://user-images.githubusercontent.com/6722691/33227981-9a35a23c-d164-11e7-9a4e-01f47e079f46.png)
![2017-11-24_22 11 52](https://user-images.githubusercontent.com/6722691/33227980-9a08336a-d164-11e7-86a6-584cdebb691b.png)
![2017-11-24_22 12 15](https://user-images.githubusercontent.com/6722691/33227979-99c6e950-d164-11e7-8974-925556e84147.png)

**Comment by T145** [comment]
Yes, definitely! Personally I'm a fan of Unity-esque "paleness" and Elucent's artistic flare, b/c they work well _being_ in MC, while still distinguishing themselves from the environment.

I'd also try to go away from the "enchanting table" inspiration w/ the cloth. If you look at the default dark infuser here: https://github.com/T145/magistics/blob/1.12.2/src/main/resources/assets/magistics/textures/blocks/infuser/dark_s.png
It has the runes that wrap around it, kind of implying it's used for more advanced things. I want the Infusers in particular to be reminiscent of their old forms. What I love about your design is how you textured that black area. If you could just somehow merge exactly what you have know w/ the old texture it would be perfect. I leave that formulation to your artistic license, and will be 100% satisfied w/ whatever v3 is. I'm only being quite nitpicky w/ this first block retexture b/c it will set the style used for the rest of the mod, so please understand I'm not trying to be a jerk.

If you want to fix up the conduit, its model, and the infuser to keep the same feel, go for it! Wait for me to update all of the other crucible variants before going for a re-texture on them. Literally I can just copy and paste that code, so it won't be long.

**Comment by Boundarybreaker** [comment]
I'm more than happy to make improvements, don't worry! I'm definitely used to going through multiple iterations of design; it's part of the process.
For now I've just made the conduit transparent so it can have TESR and you can see through it for the connector texture, and I'm probably gonna make a new look for it sometime in the future. 
![2017-11-24_23 53 41](https://user-images.githubusercontent.com/6722691/33228562-2b2d11c2-d173-11e7-90f4-def8a696d3a0.png)

To note when I make a pull request for all of this, I've made a few major changes to the build.gradle, taking out pretty much all the dependencies except for WAILA. They can definitely be put back in when they're needed, but they're currently unused and taking them out until they're necessary halves the amount of mods to load in the dev environment.

**Comment by T145** [comment]
That's perfect! If you want to adjust the Y level of the disk in the renderer go for it.

**Comment by Boundarybreaker** [comment]
I've already done it ^.^ it's at 13 bits high instead of 16 now, so it looks just fine. I've got a texture for the new light infuser, too! It's a lot closer to the original light infuser, so lemme know if you would like any tweaks.
![2017-11-25_17 11 14](https://user-images.githubusercontent.com/6722691/33236292-e2502a70-d203-11e7-9303-b76eedfe0f34.png)
![2017-11-25_17 11 18](https://user-images.githubusercontent.com/6722691/33236291-e22fc0d2-d203-11e7-9d0d-b0bd8ad6f1de.png)
![2017-11-25_17 11 09](https://user-images.githubusercontent.com/6722691/33236293-e2696ee0-d203-11e7-9e57-0f0808580f95.png)
![2017-11-25_17 11 32](https://user-images.githubusercontent.com/6722691/33236290-e1fbab76-d203-11e7-8f0c-d8290e824a91.png)

**Comment by T145** [comment]
Like I said on Discord, I love it! The only thing I'd change is the in-hand texture. It makes the infuser look like it's running in your hand. You can reference the original top side image for an idea of what to do.

**Comment by Boundarybreaker** [comment]
Ah, that's what you meant. Here's that ^.^
![t](https://user-images.githubusercontent.com/6722691/33236767-8ab5957a-d212-11e7-8f6c-64930ae33e99.png)

**Comment by Boundarybreaker** [comment]
I did a new model for the tank to give it a bit more dimensionality. No new texture for now, though. If you're happy with this, I think I'm ready to send in a PR!
![2017-11-26_13 57 07](https://user-images.githubusercontent.com/6722691/33244778-30fdf3f6-d2b2-11e7-9e0a-d07d0bee0665.png)

**Comment by T145** [comment]
Go for it! Just undo the changes to build.gradle and we're all set. Been holding off on committing changes to avoid conflicts.
